### DISCUSSION TITLE
bibliography style with arxiv link?

### DISCUSSION THREAD

**Discussion by ExpandingMan** [typst/typst#3006] [General]
Usually the thing I care about most in a bibliography are the links to arxiv.  These are usually (but, infuriatingly, not always) given by the `eprint` field in the bib entry.  None of the `typst` bibliography styles seem to print these links by default.  I always had a very similar problem with latex, which I resolve in texlive by using the `chetref` (whatever that is) bibliography style.

Is there a style that prints the arxiv links by default?  If so, does it require a specific field in the bib entry?  If not, what are the alternatives?

Thanks!

**Comment by ice1000** [comment]
I wonder exactly the same thing. The `url` entry also doesn't seem to do anything.
